app/controllers/subscriber_lists_controller.rb

```diff
@@ -37,13 +37,13 @@ def build_subscriber_list
   end
 
   def find_subscriber_list
```

Comment:
With the shorter method names you've introduced I'd be even more inclined to golf this entire method down to 3 lines, for brevity.